IdentityCore/src/webview/embeddedWebview/ui/ios/MSIDWebviewUIController.m

```diff
@@ -60,6 +60,14 @@ + (void)initialize
     });
 }
 
++ (WKWebViewConfiguration *)createWebViewConfigWithPKeyAuthUserAgent
+{
+    WKWebViewConfiguration *webConfig = [WKWebViewConfiguration new];
```

Comment:
This also needs to include preferred content mode for iOS, and have a corresponding public API in ADAL space.